\chapter[Gene regulation via histone modifications, nucleosome positioning
and nuclear architecture in \textit{P. falciparum}
]{Multiple dimensions of epigenetic gene regulation in the malaria parasite
\textit{Plasmodium falciparum}}

\graphicspath{{12_plasmodium/}}

\begin{work}

This chapter has been published in a slightly modified form in
\citep{ay:multiple} as a joint work with Ferhat Ay, Evelien Bunnik,
Jean-Philippe Vert, Karine Le Roch and William S. Noble and

\end{work}



\begin{abstract}{Abstract}

\textit{Plasmodium falciparum} is the most deadly human malaria parasite, responsible
for an estimated 207 million cases of disease and 627,000 deaths in 2012.
Recent studies reveal that the parasite actively regulates a large fraction of
its genes throughout its replicative cycle inside human red blood cells and
that epigenetics plays an important role in this precise gene regulation. Here
we discuss recent advances in our understanding of three aspects of epigenetic
regulation in \textit{P. falciparum}: changes in histone modifications, nucleosome
occupancy and the three-dimensional genome structure. We compare these three
aspects of the \textit{P. falciparum} epigenome to those of other eukaryotes, showing
that large-scale compartmentalization is particularly important in determining
histone decomposition and gene regulation in \textit{P. falciparum}. We conclude by
presenting a gene regulation model for \textit{P. falciparum} which combines the
described epigenetic factors and by discussing the implications of this model
for the future of malaria research.

Keywords: malaria, nucleosome occupancy, histone modifications,
three-dimensional genome organization, epigenetics, gene regulation, virulence
genes.
\end{abstract}

Abbreviations: PfEMP1, Plasmodium falciparum Erythrocyte Membrane Protein 1;
var, family of genes that encode PfEMP1 proteins; ApiAP2, a family of
transcription factors in Plasmodium; mRNA, messenger RNA; FISH, fluorescent in
situ hybridization; 3C, chromatin conformation capture; 4C, circularized
chromatin conformation capture; Hi-C, chromatin conformation capture coupled
to next-generation sequencing; ChIA-PET, Chromatin Interaction Analysis by
Paired-End Tag Sequencing; PTM, post-translational modification; TSS,
transcription start site; ChIP, chromatin immunoprecipitation; TAD,
topologically associated domain; H4K20me3, histone H4 lysine 20
trimethylation; H3K9ac, histone H3 lysine 9 acetylation; H3K{N}me3, histone H3
lysine {N} trimethylation; H2A, histone H2A; H2A.Z, H2B.Z, variants of histone
H2A and H2B; SHH, sonic hedgehog gene; Hox, a group of homeobox genes; OR,
olfactory receptors; hpi, hours post invasion.


\section{Introduction}


\begin{figure}
\includegraphics[width=\linewidth]{figures/fig1.png}
\caption{Overview of the {\em P. falciparum}}
\label{fig:overview}
\end{figure}

The complex life cycle of \textit{Plasmodium falciparum} includes multiple stages in
both the human host and the mosquito vector (reviewed in
\citet{greenwood:malaria}) (Fig~\ref{fig:overview}). Human
infection starts with the bite of an infected female Anopheles mosquito,
resulting in the transfer of sporozoites that quickly migrate to the liver.
Inside liver cells (hepatocytes), these sporozoites multiply extensively over
a period of approximately two weeks and are then released into the bloodstream
in the form of thousands of merozoites (Fig.~\ref{fig:overview} - liver stage).
During the next
stage of its life cycle, the parasite replicates in red blood cells
(erythrocytes) by means of an unusual process of cell division called
schizogony. While the parasite progresses through three distinct developmental
stages (ring, trophozoite and schizont), it undergoes multiple rounds of
nuclear replication followed by division of the multinucleated parasite into
16 to 32 daughter merozoites (Fig.~\ref{fig:overview} - asexual cycle).
Upon bursting out of
the host cell, these merozoites are released into the bloodstream and will
invade new erythrocytes. During the asexual cycle, the parasite can commit to
sexual development (reviewed in \citet{baker:malaria}),
resulting in differentiation into a male
or female gametocyte (Fig.~\ref{fig:overview} - sexual stage). The uptake of mature gametocytes
by a feeding mosquito followed by the further development of the parasite in
the mosquito midgut completes the \textit{P. falciparum} life cycle
(Fig.~\ref{fig:overview} - mosquito
stage).

The asexual replication cycle is responsible for symptomatic disease and for
the complications that are associated with severe malaria, such as anemia due
to rupturing of red blood cells. In addition, severe disease can result from
cytoadherence, the attachment of \textit{P. falciparum}-infected erythrocytes to the
smallest blood vessels, preventing clearance by the spleen and causing organ
dysfunction. This cytoadherence is mediated by a family of parasite virulence
proteins that are expressed on the erythrocyte surface, \textit{Plasmodium
falciparum}
Erythrocyte Membrane Protein 1 (PfEMP1) \citep{baruch:cloning, smith:switches,
su:large}. Each \textit{P. falciparum} parasite has
approximately 60 different PfEMP1 variants encoded by var genes, only one of
which is expressed at any time. Switching var gene expression enables the
parasite to escape from host immune responses \citep{bull:parasite,
roberts:rapid}. This process of antigenic
variation is one example of the excellent adaptation of the parasite to
survive in the human host.

The development of \textit{P. falciparum} through the different stages of its life
cycle is thought to be driven by coordinated changes in gene expression. Over
the last decade, it has become clear that the parasite relies on an unusual
combination of regulatory mechanisms for gene expression, and that these
mechanisms are largely dependent on epigenetic processes (reviewed in
\citep{cui:chromatin-mediated, duffy:role, hoeijmakers:placing,
horrocks:control, deitsch:mechanisms, voss:epigenetic}).
In higher eukaryotes, gene expression is often mediated by transcription
factors that bind to cell- or tissue-specific promoters and give rise to the
expression of a subset of genes specific to that cell type or tissue
\citep{dunham:integrated}.
However, despite extensive computational searches, relatively few
transcription factors have been identified in \textit{P. falciparum}
\citep{balaji:discovery, coulson:comparative}, only a
handful of which are known to be specific to a certain stage
\citep{campbell:identification}. A notable
example is PfAP2-G, a member of the ApiAP2 transcription factor family, that
drives expression of gametocyte-specific genes and is crucial for the
development of gametocytes \citep{kafsack:transcriptional, sinha:cascade}.
On the other hand, a relatively large
number of genes are predicted to encode proteins involved in chromatin
structure, mRNA decay and translation rates \citep{coulson:comparative},
suggesting that alternative
mechanisms of gene regulation, at the epigenetic as well as post-translational
levels, may be more important for gene regulation in \textit{P. falciparum.}
Here we focus on three important aspects of epigenetic gene regulation in
\textit{P.
falciparum}, all of which are related to how DNA is packed in the nucleus (see
\citet{chung:post-translational, leroch:genomics, suvorova:transcript,
kramer:rna, bunnik:polysome}
 for articles discussing post-transcriptional regulation and see
 \citet{ponts:genome-wide}
for a discussion on DNA methylation, which is not well-characterized in
\textit{P.
falciparum}). Similar to other eukaryotes, \textit{P. falciparum }packages its DNA in
the form of a condensed DNA-protein complex called chromatin. The basic
packaging unit is a nucleosome, a stretch of approximately 147~bp of DNA
wrapped around a core of eight histone proteins. Several layers of
higher-order compaction of these strings of nucleosomes together create a
highly structured nucleus. The organization of chromatin at both local and
global levels is known to be involved in transcriptional regulation
\citep{jenuwein:translating,zentner:regulation, nora:segmental,
belmont:large-scale}.
Local chromatin structure encompasses two main regulatory processes: the
post-translational modification (PTM) of histone proteins that form
nucleosomes, and nucleosome occupancy, which comprises the location,
frequency, binding strength and protein composition (i.e., variant versus
canonical histones) of nucleosomes on DNA.

At the global level, the organization of chromatin has been studied
extensively, initially using gene-by-gene approaches such as immunofluorescent
microscopy and fluorescent in situ hybridization (FISH) and, more recently,
with chromatin conformation capture (3C)-based next-generation sequencing
assays. 3C-based assays have enabled genome-wide profiling of chromatin
contacts for various organisms including human, mouse, fruit fly, budding
yeast and  \textit{P. falciparum} \citep{ay:three-dimensional,
dixon:topological, duan:three, lemieux:genome-wide,
lieberman-aiden:comprehensive, sexton:three-dimensional}. These profiles have
yielded significant insights into the relation between chromatin organization
and transcription, revealing for example the compartmentalization of the
genome into regions of transcriptionally active euchromatin and
transcriptionally silent heterochromatin. Furthermore, for the haploid
\textit{P. falciparum} genome, the 3D models inferred from these contact
profiles allowed tracking changes in nuclear organization throughout different
stages of the parasite life cycle \citep{ay:three-dimensional}.

In the following sections, we provide an overview of our current understanding
of chromatin organization and its role in transcriptional regulation in
\textit{P. falciparum}. We first describe various characteristics of local
chromatin structure and subsequently focus on three-dimensional genome
architecture. Finally, we combine these local and global views of chromatin to
provide a model that explains our current understanding of the overall nuclear
organization in \textit{P. falciparum} and the role of the epigenome in
regulating gene expression.

\section{Histone modification landscape of the \textit{P. falciparum} genome favors
euchromatin}

\subsection{Post-translational modification of histone proteins}

\begin{FPfigure}
\includegraphics[width=\linewidth]{figures/fig2.png}
\caption{
Large-scale depletion of the transcriptionally permissive histone variant
H2A.Z and activating histone marks in the telomeric cluster
visualized on the 3D \textit{P. falciparum} genome. ChIP-seq data from Bartfai et al.
\citet{bartfai:h2az} for four histone variants or marks were downloaded from
GEO (accession number: GSE23787) and mapped to the \textit{P. falciparum} genome
(PlasmoDB v9.0) using the short read alignment mode of
BWA (v0.5.9) \citep{li:fast} with default parameter settings.
Reads were post-processed,
and only the reads that map uniquely with a quality score
above 30 and with at most two mismatches were retained for further analysis.
Retained reads were subjected to PCR duplicate elimination
and then were aggregated for each non-overlapping 5~kb bin across the \textit{P.
falciparum} genome. The number of reads for each 5~kb bin was
normalized using the overall sequencing depth of the corresponding ChIP-seq
library. Plotted are the log2 ratios of sequence-depth
normalized number of reads from the ChIP-seq library versus the corresponding
input library (red: depletion, blue: enrichment) for \textbf{A:} H2A at
40 hours post invasion (hpi), \textbf{B:} H2A.Z at 10 hpi, \textbf{C:} H2A.Z
at 30 hpi, \textbf{D:} H2A.Z
at 40 hpi, \textbf{E:} H3K9ac at 40 hpi, and \textbf{F:} H3K4me3 at 40 hpi. 3D
models for the ring, trophozoite and schizont stages were generated in
\citet{ay:three-dimensional} and were colored with ChIP-seq enrichment/depletion
from 10, 20, and 40 hpi, respectively. Light blue and white spheres indicate
centromeres and telomeres, respectively. The black dashed circle
denotes the telomeric cluster for each stage. See Supporting information or
http://noble.gs.washington.edu/proj/plasmo-epigenetics for the
rotating 3D figure of each available ChIP-seq library.}
\label{fig:structure}
\end{FPfigure}

Histone proteins consist of a globular core structure and an N-terminal tail
that protrudes from this core domain. Many amino acid residues in the core
domain and in particular in the N-terminal tail can be chemically modified,
with various effects on chromatin organization (Fig.~\ref{fig:structure}). In general, the
addition of an acetyl group neutralizes the positive charge of histone
proteins and thereby disrupts the stability of the DNA-histone interaction.
This destabilization results in a more open chromatin structure and promotes a
transcriptionally permissive state. On the other hand, methylations are
uncharged and do not directly interfere with the interaction between histones
and DNA. Rather, methylations mostly function by recruiting other effector
molecules to the locus, resulting in further modifications of the chromatin.

\subsection{Activating histone marks are abundant and broadly distributed}
\afterpage{
\thispagestyle{empty}
\vspace{-1em}
\begin{table}
\renewcommand{\arraystretch}{1.5}
\begin{center}
\begin{tabular}{lp{0.35\linewidth}p{0.35\linewidth}}
\hline
\emph{Histone PTM/variant} & \emph{Other eukaryotes} & \emph{P. falciparum} \\
\hline
H3K4me3 & Promoters of active genes {\small \citep{bernstein:genomic,
kim:high-resolution, wang:combinatorial, barski:high-resolution}} & Widely
distributed in intergenic regions {\small \citep{bartfai:h2az,
salcedo-amaya:dynamic}} \\
H3K9ac & Promoters of active genes {\small \citep{wang:combinatorial,
nishida:histone}} & Widely distributed in intergenic regions {\small
\citep{bartfai:h2az, salcedo-amaya:dynamic}} \\
H3K4me3 & Promoters of active genes {\small \citep{bernstein:genomic,
kim:high-resolution, wang:combinatorial, barski:high-resolution}} & Widely
distributed in intergenic regions {\small \citep{bartfai:h2az,
salcedo-amaya:dynamic}}\\
H3K9ac & Promoters of active genes {\small \citep{wang:combinatorial,
nishida:histone}} & Widely distributed in intergenic regions {\small
\citep{bartfai:h2az, salcedo-amaya:dynamic}} \\
H3K9me3 & Silent genes {\small \citep{wang:combinatorial,
barski:high-resolution}} &
 Repressed var genes {\small \citep{lopez-rubio:genome-wide,
 chookajorn:epigenetic, lopez-rubio:5}} \\
H3K27me3 & Promoters of silent/poised genes {\small \citep{wang:combinatorial,
barski:high-resolution, mikkelsen:genome-wide}},
 absent in yeast {\small \citep{lachner:trilogies}} & Not detected {\small
 \citep{trelle:global}} \\
H3K36me3 &
Enriched in pericentromeric heterochromatin {\small \citep{chantalat:histone}};
Transcribed regions of active genes {\small \citep{wang:combinatorial,
barski:high-resolution}} &
TSS of repressed var genes {\small \citep{jiang:pfsetvs}};
3’ end coding region active genes {\small \citep{jiang:pfsetvs}} \\
H4K20me3 &
Silencing of telomeres, transposons and long terminal repeats {\small
\citep{barski:high-resolution, lachner:trilogies}};
inactive promoters {\small \citep{wang:combinatorial}} &
Repressed var genes {\small \citep{jiang:pfsetvs}} and broad distribution
across additional loci {\small \citep{lopez-rubio:genome-wide}} \\
H2A.Z &
Enriched in nucleosomes bordering active promoter (reviewed in
{\small \citep{zlatanova:h2az, talbert:histone}})
& Widely distributed in intergenic regions {\small \citep{hoeijmakers:h2az,
petter:h2az}} \\
H2B.Z &
Lineage-specific variants with specialized functions, for example enriched at
TSS in \textit{Trypanosoma brucei} {\small \citep{siegel:four}} &
Widely distributed in intergenic regions {\small \citep{hoeijmakers:h2az,
petter:h2az}} \\
\end{tabular}
\end{center}
\caption{Overview of most-studied histone modifications and variants in
\textit{P. falciparum} and comparison of their genome-wide distribution or
function in other eukaryotes.}
\label{table:histone_mod}
\end{table}}

In \textit{P. falciparum}, mass spectrometry experiments have identified at
least 50 different histone post-translational modifications (PTMs), including
methylation, acetylation, phosphorylation, ubiquitylation, and sumoylation
\citep{lasonder:insights, miao:malaria, treeck:phosphoproteomes,
trelle:global}. Subsequent chromatin immunoprecipitation (ChIP) studies have
given us insight into the genome-wide distribution of these histone marks in
the asexual cycle (Table~\ref{table:histone_mod}). In contrast to multicellular eukaryotes, a large
proportion of the genome in \textit{P. falciparum} is constitutively
acetylated \citep{miao:malaria, lopez-rubio:genome-wide}. An abundance of
activating marks has also been observed for other unicellular organisms, such
as \textit{Saccharomyces cerevisiae} and \textit{Tetrahymena thermophila}
\citep{garcia:organismal}. Inhibition of
histone acetyltransferase and deacetylase activity influences the expression
levels of the majority of genes and interferes with parasite growth
\citep{cui:histone, cui:cytotoxic, chaal:histone},
indicative of the importance of acetylation for regulating transcription
levels. Activating marks H3K9ac and H3K4me3 are mainly located in intergenic
regions \citep{bartfai:h2az, jiang:pfsetvs, salcedo-amaya:dynamic}.
Highly transcribed genes carry more H3K9ac marks in their
promoter \citep{bartfai:h2az}, 
and this marking extends into the 5’ coding region
\citep{salcedo-amaya:dynamic}.

\subsection{Repressive histone marks are scarce and localized to specific
regions}

Typical repressive marks, in particular H3K9me3, are almost exclusively found
in repressive clusters containing genes belonging to the virulence families,
such as var, rifin, stevor, and pfmc-2tm \citep{lopez-rubio:genome-wide,
jiang:pfsetvs, chookajorn:epigenetic, lopez-rubio:5}. Interestingly, H3K9me3
is also present at several additional loci, including the gene encoding the
gametocyte-specific transcription factor PfAP2-G
\citep{lopez-rubio:genome-wide} that is tightly
repressed during the asexual cycle. Transcription start sites of silent var
genes are also enriched for H3K36me3 \citep{jiang:pfsetvs},
while this modification is found at
equal levels inside coding regions of active and repressed var genes
\citep{jiang:pfsetvs, ukaegbu:recruitment}.
H3K36me3 is present at lower levels in the rest of the genome and is enriched
at the 3’ end of coding regions of active \textit{P. falciparum} genes, in
agreement with its role in transcriptional elongation in other eukaryotes. The
repressive mark H4K20me3 is also mainly present in var gene clusters, although
its enrichment is not as strong as for H3K9me3 and H3K36me3
\citep{jiang:pfsetvs}. On the other
hand, the single active var gene, out of \~60 family members, is enriched in
active histone marks, such as H3K9ac, H3K4me3, and H4 acetylations
\citep{jiang:pfsetvs, lopez-rubio:5}.
Finally, the repressive mark H3K27me3 has not been detected in the parasite
\citep{trelle:global}, similar to yeast. 
The \textit{P. falciparum} genome organization thus seems
unusual in that a large fraction of its chromatin is continuously in a
transcriptionally permissive state, while the formation of heterochromatin
seems to be limited to virulence and specific sexual genes.

\section{Histone variants and nucleosome occupancy are associated with gene
expression}

\subsection{Plasmodium exhibits a distinctive nucleosome landscape around
coding regions relative to other eukaryotes}

Nucleosome occupancy plays an important role in regulating gene expression by
allowing or restricting access of the transcription machinery to the DNA.
Nucleosomes are not placed uniformly along the genome, but show a distinct
distribution around coding regions \citep{brogaard:map,
buenrostro:quantitative, jansen:nucleosome, lee:high-resolution,
mavrich:nucleosome}. In yeast and higher eukaryotes,
the promoter is characterized by a nucleosome-depleted region, bordered on
either side by strongly positioned -1 and +1 nucleosomes, respectively, both
of which are enriched for the variant histone H2A.Z
\citep{raisner:histone, guillemette:variant, tolstorukov:comparative}. The +1 nucleosome
is located at a fixed distance relative to the transcription start site (TSS),
although this distance varies between organisms \citep{lee:high-resolution}. The +2, +3 and
subsequent nucleosomes form an array of nucleosomes with increasingly more
fuzzy positioning towards the 3’ end of the gene. Finally, the transcription
stop site is again demarcated by a strongly positioned nucleosome, followed by
another nucleosome-depleted region.

Nucleosome organization in \textit{P. falciparum} is similar to other eukaryotes in
several respects. First, the promoter region is depleted of nucleosomes
\citep{bunnik:DNA-encoded, ponts:nucleosome, westenberger:genome-wide},
the level of which correlates with transcriptional activity. Second,
highly expressed genes have a more open chromatin organization at their core
promoter than silent genes \citep{bunnik:DNA-encoded, ponts:nucleosome}.
However, the \textit{P. falciparum} nucleosome
landscape also exhibits a number of unusual features. Notably, the TSS is not
marked by a strongly positioned +1 nucleosome; instead, the strongest
nucleosomes are the first and last nucleosomes within the coding region
\citep{bunnik:DNA-encoded, ponts:nucleosome}. Furthermore,
telomeric repeats and subtelomeric regions that contain
the virulence gene families (var, rifin, etc) have higher nucleosome occupancy
levels than the bulk of the genome \citep{bunnik:DNA-encoded,
ponts:nucleosome, segal:genomic}. Intergenic regions, on the
other hand, contain lower nucleosome levels than coding regions
\citep{bunnik:DNA-encoded, ponts:nucleosome, segal:genomic, ponts:nucleosome2},
which is likely to be related to their extremely high AT-content (90-95\%).
AT-rich DNA is inherently inflexible, hampering the winding of DNA around the
histone core \citep{tillo:GC, segal:poly}. 
Finally, intergenic regions in \textit{P. falciparum} are
exclusively occupied by nucleosomes composed of histone variants H2A.Z and
H2B.Z \citep{hoeijmakers:h2az, petter:h2az}, 
which are thought to have adopted a specialized function in \textit{P.
falciparum} to allow nucleosome assembly in these highly AT-rich regions. These
histone variants are thus not restricted to promoter flanking nucleosomes but
have a much broader distribution.

\subsection{Nucleosome dynamics change in concordance with transcriptional
activity during the asexual cycle}

Another unconventional feature of nucleosome organization in \textit{P.
falciparum} is that nucleosome levels vary considerably during the asexual
replication cycle, in parallel with changes in transcriptional activity
\citep{bunnik:DNA-encoded, ponts:nucleosome}.
At the transcriptionally most active trophozoite stage, histone
levels decrease by approximately two-fold 
\citep{bunnik:DNA-encoded, ponts:nucleosome}. This nucleosome depletion
occurs in a genome-wide fashion and is not restricted to genes that are
expressed in the trophozoite stage. As the asexual cycle progresses into the
schizont stage, nucleosomes are re-assembled, resulting in condensation of DNA
as the parasites prepare for egress and re-invasion of a new red blood cell.
Given the correlation between nucleosome density in promoter regions and gene
expression levels, the dynamic nucleosome landscape in \textit{P. falciparum} may have
evolved to compensate for a paucity of specific transcription factors.
Interestingly, \textit{Trypanosoma brucei}, a parasite causing sleeping sickness in
humans, has also developed an unusual nucleosome landscape, where certain
combinations of canonical and variant histones mark the transcription
initiation and termination sites in its genome \citep{siegel:four}.
Reminiscent of the lack
of transcription factors in \textit{P. falciparum}, transcription factors have remained
elusive in \textit{T. brucei}, indicating that these parasites may have followed
parallel evolutionary pathways towards the use of the nucleosome landscape as
a mechanism to regulate gene expression.


\section{Three-dimensional conformation of the \textit{P. falciparum} genome}

\subsection{Principles of nuclear organization in \textit{P. falciparum}}

It has been long known that the eukaryotic nucleus is a highly structured
entity. In addition to three-dimensional conformation of the
chromatin-packaged DNA, key structural landmarks include the nuclear envelope,
nuclear pores and nucleoli. For decades, various microscopic imaging
techniques have been the “go-to” tools for understanding nuclear organization
and chromatin architecture in many different organisms
\citep{cremer:chromosome, misteli:beyond, takizawa:meaning}. In \textit{P.
falciparum}, FISH applications have been instrumental in demonstrating
important characteristics of genome organization in the parasite. In
particular, silent var genes were shown to colocalize with each other near the
nuclear periphery, while the single active var gene is located elsewhere

\citep{lopez-rubio:genome-wide, freitas-junior:frequent, ralph:antigenic}.
Together with the other epigenetic mechanisms outlined above —
histone modifications, histone variants and nucleosome occupancy — the
non-random organization of DNA into repressive centers is believed to play a
crucial role in the one-at-a-time expression of 60 genes in the var family.
Another intriguing discovery from FISH experiments was that the ribosomal DNA
loci that are distributed in a seemingly random fashion on different \textit{P.
falciparum} chromosomes show non-random colocalization in 3D
\citep{mancio-silva:clustering}. A more
recent study employed several ultrastructural microscopy techniques to study
the distribution of nuclear pore complexes and chromatin throughout the
\textit{P.
falciparum} asexual cycle \citep{weiner:3d}, demonstrating a striking increase in pore
density during the transcriptionally active trophozoite stage, as well as
chromatin decomposition near the nuclear envelope. These changes parallel
previously observed changes in transcriptional activity and nucleosome
occupancy that have been discussed above \citep{ponts:nucleosome}.

\subsection{Profiling of eukaryotic genome architecture using next-generation
sequencing applications}

Within the last decade, the field of genome architecture has been
revolutionized by breakthroughs in combining next generation sequencing with
molecular assays that measure proximities of DNA regions to certain nuclear
landmarks (e.g., lamina, nucleolus) or to other regions in cis or trans (e.g.,
4C, Hi-C, ChIA-PET) \citep{duan:three-dimensional,
lieberman-aiden:comprehensive, fullwood:oestrogen-receptor-alpha-bound,
guelen:domain, koningsbruggen:high-resolution, vogel:detection,
zhao:circular} (see \citep{steensel:genomics} for review).
Applications of these
techniques to multiple genomes including human and mouse have revealed the
organizational hallmarks of genome architecture. These include localization of
gene-rich regions near the nuclear center and heterochromatin near the nuclear
lamina \citep{guelen:domain}, colocalization of ribosomal DNA loci near
nucleoli \citep{koningsbruggen:high-resolution}, and
megabase-scale open/closed chromatin compartments
\citep{lieberman-aiden:comprehensive}. In addition, genomes
of higher eukaryotes are partitioned into megabase-sized topologically
associated domains (TADs) that are enriched for interactions within but not
across domains and are separated from each other by insulator proteins
\citep{dixon:topological, nora:spatial, sofueva:cohesin-mediated}
(see \citep{nora:segmental} 
for review). Finally, these studies have provided us with
examples of cell type-specific chromatin loops bringing distal regulatory
elements in close 3D proximity. Long-range chromatin loops that play
regulatory roles in gene expression include Hox cluster silencing
\citep{ferraiuolo:three-dimensional, rousseau:hox},
control of SHH gene by an enhancer that is located 1~Mb away in human
\citep{li:extensive} and
a validated set of cell type-specific enhancers in mouse \citep{shen:map}.

\subsection{Profiling of \textit{P. falciparum} genome architecture during the asexual
cycle}

\begin{table}
\renewcommand{\arraystretch}{1.5}
\begin{tabular}{p{0.15\linewidth}p{0.28\linewidth}p{0.28\linewidth}p{0.28\linewidth}}
\hline
\textit{Feature} & \textit{Ring} & \textit{Trophozoite} & \textit{Schizont} \\
\hline
Nuclear size & Small ($\sim$700~nm diameter) {\small \citep{weiner:3d, bannister:making}}
& Large ($\sim$700~nm diameter) {\small \citep{weiner:3d, bannister:making}}
& Small ($\sim$850~nm diameter) {\small \citep{weiner:3d, bannister:making}}
\\ Nuclear pores
& Few (3-7), clustered together {\small \citep{weiner:3d}}
& Many (12-58), uniformly distributed {\small \citep{weiner:3d}}
& Few per daughter nucleus (2-6), clustered together {\small \citep{weiner:3d}}
\\ Nucleosome occupancy
& High {\small \citep{bunnik:DNA-encoded, ponts:nucleosome}}
& Low {\small \citep{bunnik:DNA-encoded, ponts:nucleosome}}
& High {\small \citep{bunnik:DNA-encoded, ponts:nucleosome}}
\\ Chromatin compaction
& Compact {\small \citep{ay:three-dimensional, bunnik:DNA-encoded, ponts:nucleosome, weiner:3d}}
&  Open {\small \citep{ay:three-dimensional, bunnik:DNA-encoded, ponts:nucleosome, weiner:3d}}
& Compact {\small \citep{ay:three-dimensional, bunnik:DNA-encoded, ponts:nucleosome, weiner:3d}}
\\ Chromosome territories
& Conflicting reports (absent {\small \citep{lemieux:genome-wide}} vs present {\small \citep{ay:three-dimensional}})
& Partially lost {\small \citep{ay:three-dimensional}}
& Present {\small \citep{ay:three-dimensional}}
\\ Centromere locations
& Conflicting reports (colocalized {\small \citep{ay:three-dimensional}} vs dispersed {\small \citep{lemieux:genome-wide, hoeijmakers:plasmodium}})
& Colocalized {\small \citep{ay:three-dimensional, hoeijmakers:plasmodium}}
& Colocalized {\small \citep{ay:three-dimensional, hoeijmakers:plasmodium}}
\\ Telomere locations
& Colocalized near periphery {\small \citep{ay:three-dimensional, freitas-junior:frequent}}
& Colocalized near periphery {\small \citep{ay:three-dimensional, freitas-junior:frequent}}
& Colocalized near periphery {\small \citep{ay:three-dimensional, freitas-junior:frequent}}
\\ Virulence gene locations
& Colocalized {\small \citep{lemieux:genome-wide}} near periphery {\small \citep{ay:three-dimensional, lopez-rubio:genome-wide, freitas-junior:frequent}}
& Colocalized near periphery {\small \citep{ay:three-dimensional, lopez-rubio:genome-wide, freitas-junior:frequent}}
& Colocalized near periphery {\small \citep{ay:three-dimensional, lopez-rubio:genome-wide, freitas-junior:frequent}}
\\ rDNA gene locations
& Conflicting reports (all loci clustered {\small \citep{mancio-silva:clustering}} vs strong clustering of only
active loci {\small \citep{ay:three-dimensional, lemieux:genome-wide}})
& Conflicting reports (dispersed {\small \citep{mancio-silva:clustering}} vs weak clustering of only active loci
{\small \citep{ay:three-dimensional}})
& Conflicting reports (dispersed {\small \citep{mancio-silva:clustering}} vs weak clustering of only active loci
{\small \citep{ay:three-dimensional}}) \\
\end{tabular}
\caption{Summary of organizational features of \textit{P. falciparum} nucleus and
genome at three distinct stages during asexual parasite replication in human
red blood cells (asexual cycle).}
\label{table:features}
\end{table}

As is the case for many other next generation sequencing-based assays,
application of these genome architecture assays has been challenging for the
AT-rich genome of \textit{P. falciparum}. However, within the last year, two groups
have published their results using Hi-C, one profiling the genome architecture
of different \textit{P. falciparum} strains \citep{lemieux:genome-wide} 
and the other modeling the 3D
structure of \textit{P. falciparum}-3D7 at three key stages during its asexual
replication cycle within human red blood cells \citep{ay:three-dimensional}.
These studies revealed
key characteristics of \textit{P. falciparum} genome structure (Table~\ref{table:features}), including
colocalization of centromeres, colocalization of telomeres near the nuclear
periphery, colocalization of both internal and subtelomeric virulence gene
clusters near the telomeres, colocalization of rDNA loci that are active in
ring stage parasites and maintenance of chromosomes territories (see
\citet{ay:three-dimensional} for details). Furthermore, Hi-C profiles from
\citet{ay:three-dimensional} exhibit different
polymer behavior in the most transcriptionally active trophozoite stage
compared to the other two stages, suggesting a link between overall chromatin
compaction and transcriptional activity. The degree of telomere colocalization
and the repressive effect of the telomeric compartment is also most pronounced
in this trophozoite stage, suggesting a strict compartmentalization to
segregate genes that need to be repressed from the rest. Finally, both the
Hi-C contact maps and the 3D models inferred from them suggest a tight
correlation between the 3D location of a gene and its expression. Gene pairs
located nearby in 3D have significantly higher expression correlation compared
to other pairs, even after discarding intra-chromosomal pairs that would be
biased by their genomic distance in 1D \citep{ay:three-dimensional}. Overall, these observations
suggest that \textit{P. falciparum} chromatin is highly structured at the large scale
and that this structure provides a potential epigenetic mechanism to regulate
gene expression.

The folded chromosome structure seen in \textit{P. falciparum} is similar to what has
been observed in budding and fission yeast \citep{duan:three,
tanizawa:mapping}. However, chromosome
looping to achieve localization of var genes in repressive perinuclear
compartments results in a more complex three-dimensional organization of the
\textit{P. falciparum} genome compared to yeast, even though these organisms have
similarly sized genomes \citep{ay:three-dimensional}.
Interestingly, the clonal var gene expression
and clustering of all remaining var genes in repressive heterochromatin is
strikingly similar to the epigenetic signature of the ~1,400 olfactory
receptor genes in the mouse, all except one of which are located in
heterochromatic foci enriched for H3K9me3 and H4K20me3, resulting in monogenic
and monoallelic expression \citep{magklara:epigenetic, lyons:epigenetic}.
In comparison to higher eukaryotes, such
as human, mouse and fly, the \textit{P. falciparum} genome organization is relatively
simple and does not display TADs. The nuclear architecture in \textit{P.
falciparum}
thus exploits features from both unicellular and multicellular organisms.

\section{A combined model of epigenetic gene regulation in \textit{P.
falciparum}}

\subsection{Nuclear organization and gene regulation}

\begin{figure}
\begin{center}
\includegraphics[width=\linewidth]{figures/fig3.png}
\end{center}
\caption{\textbf{Visualization of ChIP-seq data from Jiang et al. [46] on the
3D {\em P.
 falciparum} genome at the ring stage}.
 ChIP-seq data from Jiang et al. for 5 histone marks were downloaded from SRA
 (accession number: SRP022761) and processed as described in the caption of
 Figure~\ref{fig:structure}. Due to lack of input libraries from this publication, the input
 libraries from Bartfai et al. at different time points were pooled into one
 aggregated input library which is then used for normalization of each Jiang
 et al. ChIP-seq library. Similar to Figure~\ref{fig:structure}, log2 ratios of ChIP-seq versus
 input were plotted for \textbf{A:} H3K9me3, \textbf{B:} H3K36me3, \textbf{C:}
 H4K20me3, and \textbf{D:} H3K4me3
 at 18 hpi. The 3D model for the ring stage from \citep{ay:three-dimensional} was used to
 visualize enrichment/depletion of each histone mark. See
 http://noble.gs.washington.edu/proj/plasmo-epigenetics for the rotating 3D
 figure of each available ChIP-seq library.
}
\label{fig:histone}
\end{figure}

The epigenetic makeup of the \textit{P. falciparum} genome, as outlined above, points
towards a binary nuclear organization, with the majority of the genome present
in the form of euchromatin, while a limited number of genes are organized into
strongly repressed heterochromatin. This heterochromatin is localized at the
nuclear periphery and is characterized by high nucleosome density
(Fig.~\ref{fig:structure}A),
the presence of repressive histone marks H3K9me3, H3K36me3 and H4K20me3 (Fig.
~\ref{fig:histone}A-C), and the absence of the transcription-associated histone variant H2A.Z
(Fig.~\ref{fig:structure}B-D) and histone marks H3K9ac (Fig.~\ref{fig:structure}E) and H3K4me3
(Fig.~\ref{fig:structure}F and 3D).
It was recently demonstrated that heterochromatin protein 1 (HP1) and \textit{P.
falciparum} histone deacetylase 2 (PfHda2) are both essential for maintaining
heterochromatic regions \citep{brancucci:heterochromatin, coleman:plasmodium}.
Depletion of either HP1 or PfHda2 resulted in
an arrest of parasite development at the trophozoite stage and a loss of var
gene repression. In addition, an increase in the number of parasites
differentiating into gametocytes was observed, indicating that the gametocyte
transcription factor locus pfap2-g is also under strict epigenetic control.
The remaining euchromatic fraction of the genome has several notable features,
including perinuclear compartments containing the active var gene or active
rDNA genes (Fig.~\ref{fig:final}A). In addition, clustering of silent genes that are
specific to other stages of the parasite’s life cycle
\citep{ay:three-dimensional}, suggests the
presence of small heterochromatic islands, as observed at the trophozoite
stage by advanced transmission and scanning electron microscopy
\citep{weiner:3d}.

\subsection{Remodeling of the nuclear organization during the asexual cycle}

\begin{figure}
\begin{center}
\includegraphics[width=\linewidth]{figures/fig4.png}
\end{center}
\caption{\textbf{Model for \textit{P. falciparum} epigenetic gene regulation.}
 \textbf{A:} Nuclear organization and gene regulation in \textit{P. falciparum}. Centromeric
 (dark blue) and telomeric (red) clusters are localized at the nuclear
 periphery. Subtelomeric virulence genes (blue) are anchored to the nuclear
 perimeter and cluster with internally located var genes in repressive
 center(s), characterized by repressive histone marks H3K9me3 and H3K36me3.
 The single active var gene (green) is located in a perinuclear compartment
 away from the repressive center(s). In addition, active rDNA genes (orange)
 also cluster at the nuclear periphery. The remaining genome (purple) is
 largely present in an open, euchromatic state with a number of notable
 features. (i) Nucleosome levels are high in genic and lower in intergenic
 regions, while gene expression correlates with nucleosome density at the
 transcription start site. (ii) Intergenic regions are bound by nucleosomes
 containing histone variants H2A.Z and H2B.Z. (iii) Intergenic regions contain
 H3K4me3, the level of which does not influence transcriptional activity. (iv)
 H3K9ac is mainly found in intergenic regions and extends into 5’ ends of
 coding regions, with highly expressed genes showing higher levels of H3K9ac.
 (v) Active genes are marked with H3K36me3 towards their 3’ end. \textbf{B:}
 Remodeling
 of the nuclear organization during the asexual cycle. Extensive remodeling of
 the nucleus takes place as the parasite progresses through the ring,
 trophozoite and schizont stages. In the transition from the relatively inert
 ring stage to the transcriptionally active trophozoite stage, the size of the
 nucleus and the number of nuclear pores increase, accompanied by a decrease
 in genome-wide nucleosome levels, resulting in an open chromatin structure
 that allows high transcription rates. In the schizont stage, the nucleus
 divides and recompacts, histones are re-assembled and transcription is
 shut-down, to facilitate egress of the parasites’ daughter cells and
 re-invasion of new red blood cells.}
\label{fig:final}
\end{figure}

Microarray and RNA-seq studies have shown that 70-80\% of all genes are
expressed in the asexual replication cycle, in particular during the
trophozoite stage \citep{bunnik:polysome, leroch:discovery, otto:new}.
During the 48-hour cycle, the nucleus and
chromatin are dramatically remodeled to facilitate this high transcriptional
activity (Fig.~\ref{fig:final}B and Table~\ref{table:features}). First, the nucleus expands in size
\citep{weiner:3d}, which
can also be readily observed in microscopy images of Giemsa stained parasites
\citep{ay:three-dimensional}. Second, the number of nuclear pores increases 
drastically, from 3-7
clustered pores in the ring stage to 12-58 pores that are uniformly
distributed around the nucleus in the trophozoite stage \citep{weiner:3d}.
Third, in line
with the increased nuclear volume, the chromatin opens up
\citep{ay:three-dimensional, weiner:3d}, accompanied
by removal of nucleosomes \citep{bunnik:DNA-encoded, ponts:nucleosome}
 and increased intermingling of chromosomes
\citep{ay:three-dimensional}. Despite these large-scale nuclear dynamics, the centromeres, telomeres
and repressed var genes remain clustered. The correlation of nucleosome
density of gene promoters with transcriptional activity of individual genes
suggests that local chromatin organization may play an important role in
regulating the level of gene expression \citep{bunnik:DNA-encoded}. The transitioning of the
parasite from the trophozoite stage to the schizont stage is characterized by
a reversion of nuclear changes, including reassembly of nucleosomes and
re-establishment of chromosomal territories, which results in recompaction of
the genome. Finally, during DNA replication, the nucleus divides into multiple
small daughter nuclei, each with a small number of the nuclear pores that were
present in the original nucleus \citep{weiner:3d}.

\section{Outstanding questions}

\subsection{Clustering of repressive heterochromatin}

Whether heterochromatin containing silent var genes is organized into a single
large repressive center or is divided over a small number of perinuclear foci
remains a topic of debate. FISH images visualizing the location of
telomere-associated repeat elements or var gene promoters typically show 2-6
foci distributed around the nucleus \citep{lopez-rubio:genome-wide,
freitas-junior:frequent, ralph:antigenic, voss:var}. On the other hand, single
foci were observed by immunofluorescence microscopy for H3K9me3, H3K36me3, and
heterochromatin protein 1 \citep{ukaegbu:recruitment, dahan:pfsec13},
all of which are strongly associated with
the repressed var genes. In addition, the Hi-C-derived three-dimensional
models of the \textit{P. falciparum} genome showed strong clustering of centromeres and
telomeres \citep{ay:three-dimensional} (Fig.~\ref{fig:final}A), a
chromosome configuration that has been observed in
other organisms \citep{duan:three-dimensional, tanizawa:mapping,
umbarger:three-dimensional}. These models suggested the organization of
subtelomeric var genes into a single cluster at the nuclear perimeter. Such
organization, even though seemingly contradicting the FISH data, may be due to
aggregation across a large population of cells for Hi-C experiments. If each
var gene cluster is randomly located in one of multiple repressive clusters in
each cell, then the aggregate signal would suggest colocalization of all var
genes. However, it may conceivably be beneficial to locate all repressed genes
in close proximity of each other to regulate the expression of a single var
gene and the tight repression of all remaining family members. Additional
experiments will be necessary to unravel the precise mechanisms by which var
gene expression is controlled, by further dissecting the effect of gene
localization, nuclear architecture, and gene-to-gene communication on this
process. In particular, Hi-C experiments on single cells would likely provide
significant insight into the localization of active and repressed var genes,
as well as the extent of cell-to-cell variability. 

\subsection{Mediators of epigenetic control and nuclear remodeling}

Drastic remodeling of the nucleus and chromatin are likely to be driving
forces behind the wave of transcriptional activity during the trophozoite
stage. Components involved in these dynamic processes may thus be promising
targets for antimalarial drugs. Future research should therefore focus on
understanding the molecular mechanisms involved in chromatin and nuclear
remodeling. For example, very little is known about proteins and enzymes that
regulate the formation of heterochromatin and the global nuclear architecture,
with the exception of the role of HP1 in maintaining repressive perinuclear
chromatin containing the var genes and the pfap2-g locus. A multitude of such
proteins has been identified in other organisms, most notably RNA polymerase
III-associated factor (TFIIIC), cohesin and CCCTC binding factor (CTCF)
(reviewed in \citep{gomez-diaz:architectural}), and are likely 
to have homologues in \textit{P.
falciparum}. Other
potential drug targets include key components involved in expansion of the
nuclear membrane and chromatin remodeling enzymes that regulate the global
nucleosome eviction and re-assembly during the trophozoite and schizont
stages. Analysis of chromatin-associated proteins by proteomics-based
approaches will likely identify many candidates that may be involved in these
processes. In addition, the application of novel genetic engineering tools in
\textit{P. falciparum}, such as the CRISPR/Cas9 system \citep{ghorbal:genome,
zhang:efficient, wagner:efficient}, may enable us to study
the effect of gene deletion or translocation on genome structure to better
understand the determinants of nuclear architecture.

\subsection{Epigenetic control in other parasite stages}
The epigenetic regulation model we present here is based on profiles taken
during the asexual replication cycle. During this phase of the parasite’s life
cycle, the genome seems to be largely shaped by the strict one-at-a-time
expression of the var genes. The absence of var gene expression in all other
parasite stages may have a large impact on chromatin organization. In
addition, while some genes may be constitutively expressed during the
parasite’s life cycle, others may be silenced or activated in these
alternative and highly variable stages, ranging from the male and female
gametocyte, via the diploid zygote in the mosquito midgut, to the haploid
sporozoite. Therefore, we expect generating genome-wide profiles of histone
modifications, nucleosome landscape and three-dimensional architecture during
these other parasite stages to be of great interest to further explore the
epigenetic regulatory mechanisms in \textit{P. falciparum}. 

Furthermore, we know very little about the role of epigenetic control in
transcriptional regulation in other Plasmodium species. \textit{P. vivax}, for example,
has a much lower AT-content (on average 57\%), which is likely to influence the
binding kinetics and preferences of nucleosomes. In addition, \textit{P.
vivax}
expresses a large proportion of its gene family encoding for variant surface
proteins (vir) during the blood stage \citep{bozdech:transcriptome,
fernandez-becerra:variant}. The absence of clonal
expression as seen for the var family in \textit{P. falciparum} may relieve the
requirements for strictly repressive heterochromatin in \textit{P. vivax}. Determining
the nucleosome landscape, the location of histone modifications and the
three-dimensional structure of the \textit{P. vivax} genome will therefore also be
extremely informative for our understanding of epigenetic gene regulation.


\section{Conclusions and prospects}
An increasing amount of data highlights the importance of epigenetic
mechanisms in regulating gene expression in \textit{P. falciparum} and other
eukaryotes, including human and mouse \citep{ay:three-dimensional,
dixon:topological, duan:three-dimensional, lemieux:genome-wide,
lieberman-aiden:comprehensive, sexton:three-dimensional}.
Here we have discussed multiple
layers of epigenetic control, including histone modifications, nucleosome
occupancy, histone variants and genome architecture, which are involved in the
precise gene regulation during the asexual replication cycle of the malaria
parasite, \textit{P. falciparum}. We summarized the current understanding of the
interplay among these different layers and how these layers shape the overall
nuclear organization and connect to overall transcriptional activity and to
the one-at-a-time expression of var genes.

Better characterization of epigenetic regulation in \textit{P. falciparum} will
stimulate interest in several exciting directions in malaria research. Further
studies into the establishment and maintenance of strong repressive
compartments in the nucleus may reveal the underlying regulatory mechanisms
and lead to the identification of proteins involved in this process.
Disrupting the function of proteins responsible for maintaining
heterochromatin, such as HP1 \citep{brancucci:heterochromatin},
could be an effective strategy to block
parasite replication during the asexual cycle. Another important event in the
malaria life cycle is gametocytogenesis, which was recently shown to be driven
by the transcription factor PfAP2-G \citep{kafsack:transcriptional,
sinha:cascade}. It would be interesting to fully
characterize the epigenetic factors, such as genome architecture, that help
PfAP2-G target and regulate gametocyte-specific genes. In addition to layers
of epigenetic regulation we focused on here, post-transcriptional and
translational controls are likely to be involved in the timing of protein
expression \citep{suvorova:transcript, kramer:rna, bunnik:polysome, leroch:global}.
Increased insight into these regulatory processes
would significantly advance our understanding of parasite biology and could
mark a major breakthrough in our fight against malaria.
