\section*{Introduction}
Malaria remains a major contributor to the global burden of disease, with an estimated 219 million infected individuals and 660,000 deaths annually \citep{who:malaria}. One of the main limiting factors for the development of novel therapies is our poor understanding of mechanisms regulating the parasite's complex life cycle, which involves several distinct parasitic stages in the human and mosquito hosts. Regulation of these developmental stages is thought to be controlled by coordinated changes in gene expression. In addition, virulence associated with the human malaria parasite, {\em Plasmodium falciparum}, is known to be directly linked to the parasite's ability to tightly control the expression of genes involved in antigenic variations on the surface of infected red blood cells. Some progress has been made in elucidating mechanisms controlling the expression of these virulence genes \citep{duraisingh:heterochromatin, freitas-junior:telomeric}. Furthermore, a limited number of putative sequence-specific transcription factors has been identified in the parasite genome \citep{balaji:discovery, coulson:comparative}, including 27 ApiAP2 plant-like TFs, and drastic changes in chromatin structure related to transcriptional activity have been observed throughout the parasite erythrocytic cycle \citep{ponts:nucleosome}. However, general and specific mechanisms controlling the expression of the ~6,372 parasite genes remain poorly understood.

In higher eukaryotes, several analyses have emphasized the role of genome architecture in regulating transcription. Compartmentalization of the nucleus, chromatin loops and long-range interactions contribute to a complex regulatory network \citep{homouz:3d, kalhor:genome,  lieberman-aiden:comprehensive, dixon:topological}. In {\em P. falciparum}, little is known about the effect of genome organization on gene expression. Recent data indicate that genes involved in control of parasite virulence ({\em var} genes) are associated with repressive centers at the nuclear periphery \citep{duraisingh:heterochromatin, dzikowski:mechanisms, lopez-rubio:genome-wide} and that ribosomal DNA gene clusters are also colocalized \citep{mancio-silva:clustering, lemieux:genome-wide}. However, a global picture of the nuclear architecture throughout the parasite erythrocytic cycle progression and its role in transcriptional regulation is not yet available.

Chromosome conformation capture coupled with next generation sequencing (Hi-C) measures the population average frequency of contacts between pairs of DNA fragments in 3D space and can be used to model the spatial architecture of the genome \citep{lieberman-aiden:comprehensive, duan:three, kalhor:genome}. Here, we performed a variant of the Hi-C protocol, tethered conformation capture \citep{kalhor:genome}, to model at 10 kb resolution the spatial organization of the {\em P. falciparum} genome throughout its erythrocytic cycle. Our results indicate that the {\em P. falciparum} genome is highly structured, with strong colocalization of centromeres, telomeres, active rDNA genes and virulence gene clusters. These virulence genes exhibit distinctive contact patterns and may therefore contribute to establishing the three-dimensional structure of the {\em P. falciparum} genome. We identified discrete chromosomal territories during the early and late stages of the parasite erythrocytic cycle, which are partially lost in the highly transcriptionally active trophozoite stage. Global chromosome movements during the erythrocytic cycle are coherent with levels of transcriptional activity during the different stages, and the three-dimensional genome architecture shows strong correlation with gene expression levels. Collectively, our results suggest that the {\em P. falciparum} genome organization and gene expression are strongly interconnected.

\section*{Results}
\subsection*{Assaying genome architecture of {\em P. falciparum} at three stages using Hi-C}

To study the genome architecture of {\em P. falciparum}, we harvested parasites at three stages of the infected red blood cell cycle: after invasion of red blood cells at the ring stage (0h), during high transcriptional activity at the trophozoite stage (18h) and near the end of the cycle at the schizont stage (36h), just before the newly formed parasites are released into the bloodstream. Next, we applied the Hi-C protocol \citep{kalhor:genome} with modifications to accommodate the extremely AT-rich genome of the malaria parasite (Fig.~\ref{fig:fig1}a, Supplementary Note 1, Supplementary File 1). As a control, we prepared a sample for which chromatin contacts were not preserved by crosslinking of DNA and proteins.

We evaluated the quality of the resulting data for each sample. First, we confirmed that the contact probability between two intrachromosomal loci exhibits a log-linear decay with increasing genomic distance (Fig.~\ref{fig:fig1}b, Supplementary Fig.~\ref*{suppfig:power-law}). Second, we obtained lower numbers of interchromosomal contacts from crosslinked samples relative to both random expectation and our control sample (Supplementary Table~\ref*{table:ICP}). Third, we observed that the percentage of long-range contacts (either interchromosomal or intrachromosomal $>$20 kb) was significantly higher than control and comparable to the numbers observed in yeast \citep{duan:three} (Supplementary Table~\ref*{table:ICP}). Together, these results indicated that we successfully assayed the {\em P. falciparum} genome architecture with a high signal-to-noise ratio. We then coalesced the mapped read pairs into a raw contact count matrix at 10 kb resolution, and we corrected for potential technical and experimental biases \citep{imakaev:iterative} (Fig.~\ref{fig:fig1}c, Supplementary Fig.~\ref*{suppfig:ICE}) . The resulting normalized contact maps were used to identify a subset of high-confidence contacts for each stage (Methods, Supplementary Note 2, Supplementary File 2)~\citep{ay:statistical}. We identified pairs of genes that show evidence of stage-specific contacts (Methods) and then applied gene set enrichment analysis to the set of genes that participate in such contacts.  This analysis identified significant enrichment of VRSM genes for the ring and trophozoite stages (Supplementary Table~\ref*{table:GSEAcompareStages}). This observation suggests that the proximity between some VRSM clusters changes from the ring to trophozoite stages, even though both stages show overall colocalization of VRSM clusters. A similar enrichment analysis conducted using contacts that are specific to two out of three stages resulted in no significant enrichment due to the small number of genes involved in such contacts.

Normalized contact count and confidence score matrices exhibit a canonical ``X'' shape, indicative of a folded chromosome architecture anchored at the centromere, as previously observed in yeast \citep{duan:three, tanizawa:mapping} and the bacterium {\em C. crescentus} \citep{umbarger:three-dimensional} (Fig.~\ref{fig:fig1}c-d, Supplementary Fig.~\ref*{suppfig:perChrFigs}). However, chromosomes that harbor non-subtelomeric clusters of genes involved in antigenic variation and immune evasion (Supplementary File 3; VRSM genes: {\em var}, {\em rifin}, {\em stevor} and {\em Pfmc-2tm})---chromosomes 4, 6, 7, 8 and 12---exhibit additional folding structure (Fig.~\ref{fig:fig1}c-d, Supplementary Fig.~\ref*{suppfig:perChrFigs}).

\subsection*{Three-dimensional modeling recapitulates known organizational principles of {\em Plasmodium} genome}

To better characterize the genome architecture, we generated for each stage 100 consensus 3D structures, each of which summarizes the population average (Fig.~\ref{fig:fig2}a, Methods), using multidimensional scaling (MDS) with two primary constraints \citep{duan:three}: (i) the DNA must lie within a sphere with a specified diameter \citep{bannister:making, weiner:3d} and (ii) adjacent 10kb loci must not be separated by more than 91 nm \citep{bystricky:long-range}. \emph{P. falciparum} undergoes an atypical form of cell division, resulting in schizont stage parasites with multiple independent nuclei, each containing 1n chromosomes. Note that our model assumes that a single copy of each chromosome is present in each structure, thus averaging the signal from these multiple nuclei per cell.


We performed a series of experiments to assess the robustness of our 3D inference procedure.  Our results showed only slight changes in the inferred 3D models when we varied the parameter used in conversion of contact counts to expected distances (Supplementary Table~\ref*{table:stabilityToBeta}). This was also true when we removed from the inference the two types of spatial constraints related to nuclear volume and to distances between adjacent beads (Supplementary Table~\ref*{table:stabilityToConstraints}). Finally, our experiments on the impact of the initialization step (Methods) showed that structures inferred from different initial configurations are highly similar (Supplementary Fig.~\ref*{suppfig:compareStructurePairs}), do not fall into discrete clusters (Supplementary Fig.~\ref*{suppfig:CHindices}) and all such structures exhibit common organizational hallmarks (Supplementary Fig.~\ref*{suppfig:clusteringIn100Structures}). Because of the stability of our inference procedure, hereafter we generally present and discuss the results for only one representative structure per stage.

Although the modeling procedure contains no explicit constraints on telomere or centromere locations, we observe strong colocalization of both sets of loci across all three stages (Fig.~\ref{fig:fig2}a, Supplementary Fig.~\ref*{suppfig:3Dcentromeres}, Supplementary Table~\ref*{table:witten}), with centromeres and telomeres localizing in distal regions of the nucleus. To understand further the colocalization patterns of centromeres and telomeres in each stage, we divided each chromosome into three compartments (left--mid--right or telomeric--centromeric--telomeric) using eigenvalue decomposition (Methods) and then performed hierarchical clustering on the matrix of pairwise distances between compartments (Supplementary Fig.~\ref*{suppfig:compDists}). At each stage, we observed clusters that are comprised primarily of either centromeric or telomeric compartments. In particular, during the trophozoite stage, all the centromeric compartments fall into two main clusters suggesting strong colocalization of all centromeres for this stage (Supplementary Fig.~\ref*{suppfig:compDists}d). Such strong colocalization has previously been observed by immunofluoresence microscopy at the trophozoite and schizont stages but not at the early ring stage \citep{hoeijmakers:plasmodium}. However, when the size of the nucleus is used as a marker of the parasite asexual cycle stage \citep{bannister:making, weiner:3d}, the cells that are presented as trophozoites in this previous study \citep{hoeijmakers:plasmodium} are more similar to our ring stage parasites, indicating that centromere clustering also occurs early in the erythrocytic cycle. Furthermore, if the centromeres are stochastically distributed between a small number of foci within a population, then an assay that measures average signal, such as Hi-C, will indeed demonstrate an aggregate clustering for the centromeres and not complete dispersion as suggested by a recent study \citep{lemieux:genome-wide}. These results suggest that {\em P. falciparum} nuclei are highly structured around centromeres and telomeres, consistent with known organizational principles gathered through multiple independent microscopy experiments \citep{duraisingh:heterochromatin, dzikowski:mechanisms, lopez-rubio:genome-wide, hoeijmakers:plasmodium}.


\subsection*{Virulence gene clusters on different chromosomes colocalize in 3D}

In addition to centromeres and telomeres, we observed for all VRSM gene clusters, both internal and subtelomeric, a significant colocalization with one another (Fig.~\ref{fig:fig2}a, Supplementary Table~\ref*{table:witten}). The significant colocalization for VRSM clusters as well as for centromeres and telomeres were all reproducible when we used contact counts instead of 3D distances to perform colocalization tests similar to Supplementary Table~\ref*{table:witten} (data not shown). Given colocalization of the telomeres, colocalization of subtelomeric clusters is not surprising. However, the proximity of internal VRSM clusters with one another and with subtelomeric clusters is unexpected under the random polymer looping model and, to the best of our knowledge, observed experimentally for the first time. To further validate these results inferred from our 3D models, we performed DNA fluorescence {\em in situ} hybridization (FISH) (Methods, Supplementary Note 3) on an interchromosomal pair of strongly interacting (at 10 kb resolution) VRSM clusters: the internal cluster from chromosome 7 and a subtelomeric cluster from chromosome 8. We observed strong colocalization by FISH ($>$90\% of cells, Fig.~\ref{fig:fig2}b, Supplementary Fig.~\ref*{suppfig:fish}a, Supplementary Table~\ref*{table:FISHprimers}), providing independent support for the clustering of VRSM genes. Although previous FISH results indicated that {\em var} genes form 2 to 5 clusters in 3D per cell \citep{freitas-junior:frequent, lopez-rubio:genome-wide}, others recently showed single foci for the VRSM gene-associated repressive histone mark H3K9me3 and heterochromatin protein 1 (PfHP1) \citep{dahan:pfsec13}, as well as for H3K36me3 that marks both active and silenced {\em var} genes \citep{ukaegbu:recruitment}. Because our experimental strategy (Hi-C) captures a population average, we are unable to distinguish between multiple VRSM gene clusters in 3D if the genes are randomly distributed among clusters from cell to cell. Using FISH experiments, we also observed strong colocalization ($>$90\% of cells, Fig.~\ref{fig:fig2}c, Supplementary Fig.~\ref*{suppfig:fish}b) for a pair of interchromosomal loci located outside VRSM clusters with consistent strong interactions at all three stages, while colocalization was not observed for a pair of non-interacting interchromosomal loci ($<$10\% of cells, Supplementary Fig.~\ref*{suppfig:fish}c). These results demonstrate that our population average Hi-C data agrees with a majority of single cell FISH images.

\subsection*{Highly transcribed rDNA units colocalize in 3D during the ring stage}
Similar to VRSM genes, the rDNA genes are strictly regulated during the parasite life cycle. In {\em P. falciparum}, these genes are dispersed on different chromosomes in five rDNA units containing the 18S, 5.8S and 28S genes and one repeat unit consisting of three copies of the 5S gene. A previous FISH study suggested that all rDNA units localize at a single nucleolus but also claimed that the two units on chromosomes 5 and 7 that are actively transcribed during the ring stage (A-type units) are dispersed in the ring stage \citep{mancio-silva:clustering}. However, a more recent Hi-C study of ring stage parasites demonstrated strong clustering of these two A-type units in multiple strains \citep{lemieux:genome-wide}. Analysis of our Hi-C data confirmed overall enrichment of contacts between chromosomes 5 and 7 in all three stages and showed a particular peak of enrichment centered at the rDNA unit on chromosome 5 among all interchromosomal contact partners of the rDNA unit on chromosome 7 in the ring stage (3.32x, Fig.~\ref{fig:fig3}a). We observed less striking enrichment of contacts that are not specific to or centered on the rDNA units for the other two stages (trophozoites (1.99x), schizonts (1.23x), Fig.~\ref{fig:fig3}b-c) during which the two rDNA units are not transcribed \citep{mancio-silva:clustering}. Reanalysis of the Lemieux {\em et al.} data using our processing pipeline also showed this enrichment consistently in three different NF54-derived strains in the ring stage (6.06x, 4.47x and 4.61x, respectively, Supplementary Fig.~\ref*{suppfig:rDNAunitsOn3D}a-c). Control libraries from both studies do not exhibit this enrichment (Fig.~\ref{fig:fig3}d, Supplementary Fig.~\ref*{suppfig:rDNAunitsOn3D}d). Our 3D models for the ring stage place these two A-type rDNA units near the nuclear periphery. Together with the strong colocalization between A-type rDNA, these results suggest the existence of perinuclear transcriptionally active compartments. Such compartments may play a role in separating out the single active var gene per cell from compact chromatin around (sub)telomeric regions marked by the repressive H3K9me3 modification \citep{lopez-rubio:genome-wide}. We did not observe an overall colocalization between all rDNA units in the ring stage, including the three 18S, 5.8S, 28S units and one 5S unit that are not expressed during asexual erythrocytic cycle (Supplementary Table~\ref*{table:witten}).  This observation suggests that genomic location may influence rDNA expression by the preferential colocalization of the expressed rDNA units, away from the non-expressed units.

\subsection*{Transcriptionally active trophozoite stage exhibits an open chromatin structure}
Assaying three different time points, we observed significant changes in chromatin structure throughout the erythrocytic cycle. To visualize high-level changes, we generated animations showing the movement of chromosomes as the parasite progresses through its cell cycle (Supplementary Files 4-18). We then characterized global chromatin changes by analyzing the relationship between contact frequency and genomic distance (Fig.~\ref{fig:fig1}b, Supplementary Fig.~\ref*{suppfig:power-law}). The gradient of the log-linear fit is very close to -1 in both the ring and schizont stages (-0.98 and -0.96, respectively) indicative of a fractal globule genome architecture that is usually found in higher eukaryotes \citep{lieberman-aiden:comprehensive}. Intriguingly, the intermediate and most active transcriptional stage yields a log-linear fit value with gradient -1.14, a value between the fractal (-1) and the equilibrium globule (-1.5) model suggested in yeast \citep{fudenberg:higher-order} and indicative of more chromosomal intermingling. Indeed, a value of -1.17 has been demonstrated to correspond to a state of ``unentangled rings'' similar to the fractal globule state, in which the rings may correspond to long chromosomal regions looped on or anchored to a nuclear scaffold \citep{vettorel:statistics}. It is important to note that the value of the gradient is determined solely by Hi-C contact counts and,  therefore, the above mentioned difference is independent of our 3D modeling and the change in the nuclear radius from one stage to another. Furthermore, the difference in the gradient value for trophozoites compared to the two other stages is consistent for each chromosome, suggesting that all chromosomes change their folding behavior during the trophozoite stage (Supplementary Table~\ref*{table:scalingFactors}).

In order to further investigate whether trophozoites show a more open chromatin structure than the two other stages, we systematically compared our data across all three stages. First, we computed and compared intra and interchromosomal contact probabilities for each stage (Supplementary Fig.~\ref*{suppfig:intraVSinter}). We observed that intrachromosomal contacts, even at very large distances, are more prevalent than interchromosomal contacts for all three stages, suggesting the existence and preservation of chromosome territories throughout the erythrocytic cycle. However, the enrichment in intrachromosomal contacts was the lowest for trophozoite stage for distances above 300~kb, suggesting a relative loss of territories in this stage compared to the other two. Second, we quantified how preserved the chromosomal territories are at each stage by estimating the degree of chromosome intermingling in our 3D models. We randomly sampled small spheres in the nucleus and asked, for each chromosome {\em i}, what percentage of the spheres that contain any locus from chromosome {\em i} also contain a locus from another chromosome {\em j}. Our results using different sphere sizes, and controlling for the varying nuclear diameter, consistently exhibited the highest amount of intermingling for the trophozoite stage and the highest territory preservation for the schizont stage (Supplementary Fig.~\ref*{suppfig:territory}).

To understand the architectural dynamics responsible for the systematic changes in chromatin compaction, we computed the relative movements among chromosome compartments during the erythrocytic cycle. Despite the increase in nuclear volume, many interchromosomal compartment pairs came closer together in the transition from the ring to trophozoite stage (Supplementary Fig.~\ref*{suppfig:compMovement}a, red color). Subsequently, most interchromosomal compartments moved away from each other in the transition to the schizont stage (Supplementary Fig.~\ref*{suppfig:compMovement}b, blue color), resulting in more compact chromatin that favors formation of chromosome territories. These results are consistent with a previously proposed model, in which the {\em P. falciparum} nucleus exhibits a more open chromatin configuration at the trophozoite stage, enabling interchromosomal contacts and high levels of transcriptional activity \citep{ponts:nucleosome}.

\subsection*{{\em Plasmodium} genome architecture cannot be explained by volume exclusion}
We next assessed whether the primary architectural features in {\em P. falciparum} arise from a population of constrained but otherwise random configurations of chromatin following a simple volume exclusion (VE) model, as recently shown for {\em Saccharomyces cerevisiae} \citep{tjong:physical}. We therefore repeated the Tjong {\em et al.} simulations using the same set of constraints and successfully recovered the strong correlation between the simulated map and the experimentally observed yeast contact map (raw correlation of 0.91; normalized correlation of 0.57; Fig.~\ref{fig:fig4}a, Methods, Supplementary Note 4, Supplementary Fig.~\ref*{suppfig:VEconvergence}). In contrast, our simulations for the ring, trophozoite and schizont stages of {\em P. falciparum} yielded markedly lower correlations (normalized correlation of 0.34, 0.39 and 0.49, respectively) and strikingly different contact maps compared to the experimentally observed maps (Fig.~\ref{fig:fig4}b). One significant reason for the observed discrepancy between yeast and {\em P. falciparum} is the lack of structure around clusters of VSRM genes in the simulated data (Fig.~\ref{fig:fig4}b). Accordingly, we conclude that the simple volume exclusion model, which so convincingly explains the yeast genome architecture, is insufficient to explain the observed architecture of {\em P. falciparum} genome, highlighting the need for a genome-wide assay such as Hi-C to obtain accurate structural models.

\subsection*{VRSM gene clusters form domain-like structures}
Our results from the volume exclusion modeling and from visual inspection of the contact maps suggest that the internal VRSM gene clusters are associated with distinctive structural features. All eight of the internal VRSM clusters induce a striking cross-like shape, both in the contact count and 3D distance matrices (Fig.~\ref{fig:fig5}a-b, Supplementary Fig.~\ref*{suppfig:perChrFigs}). Quantification of this phenomenon revealed a consistent contact pattern across all eight internal VRSM clusters (Supplementary Fig.~\ref*{suppfig:TADs}), suggesting that VRSM gene clusters adopt a compact, domain-like structure. Although these domain-like structures resemble topologically associated domains (TADs) described in mammals \citep{dixon:topological, nora:spatial}, the VSRM domains are much smaller (10--50 kb) compared to TADs (0.1--1 Mb). Furthermore, because VRSM genes have no orthologs in human and mouse, mechanisms regulating these domain-like structures likely differ from the one in mammalian genomes. Further understanding of how these VRSM domains are formed in  \emph{Plasmodium} would shed light on genome architecture associated regulation of VRSM gene expression.

Another interesting pattern involving internal VRSM clusters emerged from further inspection of chromosome compartments.  Five of the eight internal VRSM clusters (two on chromosome 4, one on chromosome 7 and both clusters on chromosome 12) occur at compartment boundaries (third and fourth rows of Supplementary Fig.~\ref*{suppfig:perChrFigs}). This striking overlap  suggests that VRSM genes may contribute to or rely upon the boundaries of chromosomal compartments. Taken together with the domain-like structures around these VRSM clusters, these results confirm that genome architecture is likely to be involved in the strict regulation of virulence genes during the erythrocytic cycle.

\subsection*{Expression is highly concordant with 3D localization for {\em Plasmodium} genes}
Next, we investigated the relationship between the three-dimensional genome structure and gene expression using four published expression data sets \citep{leroch:discovery, lopez-barragan:directional, otto:new, bunnik:polysome}. First, we observed that, for each of the three stages, interchromosomal pairs of genes that strongly interact (contact counts within the top 20\%) as well as gene pairs that are in close proximity ($<$20\% of the nuclear diameter) showed more correlated expression profiles than genes that are far apart (Fig.~\ref{fig:fig6}a,b), as previously observed in yeast \citep{homouz:3d}. To assess whether these observed trends are confounded by similarly expressed VRSM genes that strongly interact with each other and are placed together near telomeres by our 3D model, we repeated the above analyses by excluding all VRSM genes (Supplementary Fig.~\ref*{suppfig:expVSdistWithoutVRSM}). Even though the observed trends are weakened by exclusion of VRSM genes, the decrease in 3D distance and increase in contact count with increasing expression correlation remained significant (Supplementary Fig.~\ref*{suppfig:expVSdistWithoutVRSM}). It is also important to note that, for these analyses, we excluded intrachromosomal gene pairs to only focus on the relationship between 3D proximity and gene expression by eliminating the confounding effect caused by genes that lie nearby on a chromosome and show similar expression profiles. Second, we analyzed gene expression in relation to the repressive subtelomeric clusters \citep{duraisingh:heterochromatin, dzikowski:mechanisms, lopez-rubio:genome-wide} and other nuclear landmarks. The subset of genes that lie within 20\% of the nuclear diameter to the centroid of the telomeres showed significantly lower expression levels than more distal genes (Fig.~\ref{fig:fig6}c). The repressive effect of the subtelomeric clusters is apparent in all three stages and is strongest at the trophozoite stage, in which subtelomeric VRSM clusters are known to be tightly repressed \citep{chen:developmental}. If we remove the VRSM genes from the analysis, the repressive effect is still significant at the trophozoite stage, which is known to be the most active transcriptional stage of the erythrocytic cycle (Supplementary Fig.~\ref*{suppfig:distToCenter}a,b). Similar analysis showed higher expression levels for genes located near the nuclear center, as well as for genes close to the centroid of the centromeres (Supplementary Fig.~\ref*{suppfig:distToCenter}c,d). Furthermore, we observed significant and consistent colocalization across all three stages for 11 of the 15 expression clusters identified in \cite{leroch:discovery} (Supplementary Table~\ref*{table:witten}). Strikingly, the trophozoite stage showed significant colocalization for clusters associated with genes that are repressed during this stage (clusters 1, 3, 4, and 13-15) as well as genes that exhibit high levels of expression (clusters 6, 9, 10, and 12), confirming the strong relationship between 3D location and gene expression.

To further explore the relationship between gene expression and 3D structure, we employed an unsupervised learning method known as {\em kernel canonical correlation analysis} (kCCA) \citep{bach:kernel}. This methodology identifies a set of orthogonal gene expression profiles that exhibit coherence with respect to the 3D structure (Methods). For all stages, the projection of gene expression patterns onto the first extracted profile exhibits a striking transcriptional gradient across the 3D structure, from the telomere cluster to the opposite side of the nucleus  (Fig.~\ref{fig:fig6}c, Supplementary Fig.~\ref*{suppfig:kCCAsecond}a,c,e). The coherence with 3D structure drops significantly in the second component of the kCCA (Supplementary Fig.~\ref*{suppfig:kCCAsecond}b,d,f), suggesting that gene expression is strongly influenced by distance to the subtelomeric repressive center. To further interpret the kCCA results we employed gene set enrichment analysis \citep{subramanian:gene} on the ranked lists of projections onto the first kCCA component. The results showed, for all three stages, significant enrichment (q-value $<$ 0.01) of gene sets related to antigenic variation and translation (i.e. ribosome proteins) on the telomeric and non-telomeric side, respectively, of the extracted kCCA expression profile (Supplementary Tables~\ref*{table:RingsFirstPro},~\ref*{table:TrophsFirstPro},~\ref*{table:SchizontsFirstPro}). Similar to the colocalization test results for expression clusters of \cite{leroch:discovery}, clusters of genes that are repressed (clusters 4, 13, and 14) and expressed (clusters 6 and 9-12) in the trophozoite stage showed consistent enrichment in the strongest kCCA profile (Supplementary Table~\ref*{table:kCCAforClusters}). In addition, genes exclusively expressed in sporozoites (cluster 1) and gametocytes (clusters 3) were also strongly enriched, indicating that the repression of these genes during the asexual erythrocytic cell cycle may be related to their localization within the nucleus. Finally, for GO terms related to parasite invasion (rhoptry, myosin complex, motor activity; q-value $<$ 0.1) and for the cluster of invasion genes (cluster 15), we observed an enrichment relative to the second kCCA component, suggesting that expression of invasion genes may also be regulated by the 3D genome structure (Supplementary Tables~\ref*{table:kCCAforClusters},~\ref*{table:secondPro}).


\section*{Discussion}
This study presents the first analysis of genome architecture during the cell cycle of a eukaryotic pathogen. Overall, our data demonstrate that the genome of {\em P. falciparum} exhibits a higher degree of organization than the similarly sized budding yeast genome. Although localization of chromosomes within the {\em P. falciparum} nucleus is partially dictated by size constraints, the simple volume exclusion model observed in yeast is insufficient to explain the 3D architecture of the {\em P. falciparum} genome. In particular, a striking spatial complexity is added by clusters of virulence genes, which function as critical structural elements that shape the genome architecture. Furthermore, our model correlates well with expression levels of parasite-specific gene sets and shows strong clustering of repressed genes and highly transcribed rDNA units, indicative of a non-random genomic organization that contributes to gene regulation during the asexual erythrocytic cycle. Considering the strong association between nuclear architecture and gene expression as well as the observed domain-like structures, {\em Plasmodium} species may be excellent model organisms to study the impact of genome structure on gene regulation. The lower complexity of genome organization in organisms with similarly sized genomes, such as yeast, may indeed be less informative for such investigations.

Assaying multiple time points during the parasite's erythrocytic cycle revealed intriguing changes in genome structure between the different developmental stages. Our results show that the genome adopts a more open conformation during the trophozoite stage consistent with high transcriptional activity in this stage of the erythrocytic cycle, followed by compaction of chromosomes into discrete chromosome territories before re-invasion of a new host cell. A similar pattern was observed previously for nucleosome occupancy, with strong histone depletion at the trophozoite stage and nucleosome replacement at the schizont stage \citep{ponts:nucleosome}. Based on these observations, we hypothesize that the spatial genome organization of {\em P. falciparum}, coupled with its dynamic chromatin structure, acts as an important alternative mechanism of transcriptional regulation, possibly compensating for the lack of a diverse collection of specific transcription factors \citep{balaji:discovery, coulson:comparative} and the low capacity of the parasite to regulate gene expression in response to metabolic stress \citep{ganesan:genetically, leroch:systematic}. These changes in genome architecture could mainly be indicative of differences between the various developmental stages of the parasite, but could also be related to cell cycle progression itself. Given the importance of nuclear architecture for regulation of gene expression, disruption of its genome organization is likely to interfere with parasite development through the erythrocytic cycle and could therefore be lethal to the parasite. Compounds targeting proteins involved in establishing and maintaining the three-dimensional genome structure in {\em P. falciparum} may thus have potent antimalarial activity.

A recently published Hi-C study suggested that chromosomal territories are absent in the ring stage parasites, especially for larger chromosomes \citep{lemieux:genome-wide}. In contrast, our data provides multiple lines of evidence for the existence of chromosome territories throughout the erythrocytic cell cycle. In particular, we observed that intrachromosomal contacts, even at very large distances, are more prevalent than interchromosomal contacts. This observation is supported by our own Hi-C data in three stages as well as by our reanalysis of the Lemieux {\em et al.} data (Supplementary Fig.~\ref*{suppfig:intraVSinter}b-e). The difference between the two analyses can be traced to our improved method for discretizing the genomic distance axis, which avoids bins with few observations and, hence, high variance (Supplementary Fig.~\ref*{suppfig:intraVSinter}a versus b). Even though further experiments may be necessary to reconcile these differences, our results strongly suggest that {\em P. falciparum} chromosomes occupy distinct territories, similar to other eukaryotic genomes.

Clustering of virulence gene families into a distinct nuclear compartment is likely to play an important role in the formation of repressive heterochromatin that controls the silencing of these genes. Heterochromatin around virulence genes is characterized by histone modifications H3K36me3 \citep{jiang:pfsetvs} and H3K9me3 \citep{duraisingh:heterochromatin, lopez-rubio:genome-wide}, both of which were shown to be essential for maintaining {\em var} gene repression. The formation of heterochromatin is directed by the interaction of PfSIP2 with specific DNA motifs in promoters of virulence genes and in subtelomeric domains \citep{flueck:major}, but additional factors are likely to contribute to this process. The question remains, however, how the formation of this repressive center is regulated and whether the colocalization of virulence gene clusters is a cause or a consequence of their transcriptional silencing. One experiment that would shed light on this issue would be to relocate a {\em var} gene to a different location in the genome and to monitor how the introduction of this novel {\em var} gene locus influences genome structure, although technical challenges that come with manipulation of the {\em P. falciparum} genome may prevent such procedures. Virulence genes are expressed on the surface of red blood cells and are therefore important antigens for the humoral immune system. A better understanding of virulence gene silencing will provide us with more opportunities to interfere with this process, which would ultimately benefit vaccine development.

In this study, we modeled the {\em P. falciparum} genome architecture based on the average signal from a population of parasites. However, it can be expected that considerable variability in genome conformation exists from cell to cell, as recently demonstrated in mouse \citep{nagano:single-cell}. While challenging, it would be interesting to perform Hi-C analysis on individual parasites to reveal the extent of inter-cellular variation in {\em P. falciparum} genome architecture. This experiment would also allow a more detailed analysis of the clustering of {\em var} genes in one or multiple repressive centers, as well as the differential localization of the single active {\em var} gene.

In conclusion, this study demonstrates the unique role of genome organization in transcriptional regulation in the human malaria parasite. In other eukaryotes such as human and mouse, genome organization has been shown to participate in gene regulation through formation of specific chromatin loops that bring enhancers and enhancer-like elements in proximity to their target promoters. However, a global reorganization of the entire genome correlated with changes in transcriptional capacity, as described here for {\em P. falciparum}, has not been observed for any of the genomes studied so far. Therefore, our data proposes a novel mechanism of gene regulation for {\em P. falciparum} that can operate without relying on specific transcription factors or enhancer elements. Similar to other eukaryotes, gene expression in {\em P. falciparum} is likely to be regulated by multiple layers of control at both transcriptional and translational levels. However, the necessity to transcriptionally repress distinct groups of parasite-specific genes may have driven {\em P. falciparum} to adopt this exceptional genome organization.

