\documentclass[10pt]{article}
\usepackage[margin=1in]{geometry}
\usepackage{graphicx}
\usepackage{url}
\usepackage{comment}
\usepackage{float}
\usepackage{subfig}
\usepackage{multirow}
\usepackage{amssymb}
\usepackage{color}
%\usepackage[numbers,round,authoryear]{natbib}
\usepackage{hyperref}
\usepackage{comment}
\providecommand{\e}[1]{\ensuremath{10^{#1}}}

\usepackage[table,svgnames]{xcolor}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage{natbib}

\newcommand{\qed}{\hfill \mbox{\raggedright \rule{.07in}{.1in}}}
\newcommand{\remove}[1]{}

\renewcommand{\tablename}{\bfseries Table}
\renewcommand{\figurename}{\bfseries Figure}

% External reference.
\usepackage{xr}
\newcommand{\bigcell}[2]{\begin{tabular}{@{}#1@{}}#2\end{tabular}}
% Allow references to the supp tables and figures.
\externaldocument{SI}
%\externaldocument[m-]{methods/methods}

% author block
\usepackage{authblk}

% LIMITED TO 10 words (or 90 characters).
\title{Three-dimensional modeling of the \emph{P. falciparum} genome during the erythrocytic cycle reveals a strong connection between genome architecture and gene expression}

\author[1]{Ferhat Ay\thanks{These authors contributed equally and are listed alphabetically.}}
\author[2]{Evelien M. Bunnik$^{*}$}
\author[3,4,5]{Nelle Varoquaux$^{*}$}
\author[2]{Sebastiaan M. Bol}
\author[2]{Jacques Prudhomme}
\author[3,4,5]{Jean-Philippe Vert}
\author[1,6]{William Stafford Noble$^{\dag}$}
\author[2]{Karine G. Le Roch\thanks{Corresponding authors: william-noble@uw.edu and karine.leroch@ucr.edu}}

\scriptsize{
\affil[1]{Department of Genome Sciences, University of Washington, Seattle, WA, USA, 98195}
\affil[2]{Department of Cell Biology and Neuroscience, University of California, Riverside, CA, USA, 92521}
\affil[3]{Centre for Computational Biology, Mines ParisTech, Fontainebleau, F-77300 France}
\affil[4]{Institut Curie, Paris, F-75248, France}
\affil[5]{U900, INSERM, Paris, F-75248, France}
\affil[6]{Department of Computer Science and Engineering, University of Washington, Seattle, WA, USA, 98195}
}

%\thanks{ferhatay@uw.edu}
%\thanks{t.bailey@imb.uq.edu.au}
\renewcommand\Authands{ and }
\date{}

\begin{document}
\maketitle
\begin{abstract}
The development of the human malaria parasite {\em Plasmodium falciparum} is controlled by coordinated changes in gene expression throughout its complex life cycle, but the corresponding regulatory mechanisms are incompletely understood. To study the relationship between genome architecture and gene regulation in {\em Plasmodium}, we assayed the genome architecture of {\em P. falciparum} at three time points during its erythrocytic (asexual) cycle. Using chromosome conformation capture coupled with next-generation sequencing technology (Hi-C), we obtained high-resolution chromosomal contact maps, which we then used to construct a consensus three-dimensional genome structure for each time point. We observed strong clustering of centromeres, telomeres, ribosomal DNA and virulence genes, resulting in a complex architecture that cannot be explained by a simple volume exclusion model. Internal virulence gene clusters exhibit domain-like structures in contact maps, suggesting that they play an important role in the genome architecture. Midway during the erythrocytic cycle, at the highly transcriptionally active trophozoite stage, the genome adopts a more open chromatin structure with increased chromosomal intermingling. In addition, we observed reduced expression of genes located in spatial proximity to the repressive subtelomeric center, and colocalization of distinct groups of parasite-specific genes with coordinated expression profiles. Overall, our results are indicative of a strong association between the {\em P. falciparum} spatial genome organization and gene expression. Understanding the molecular processes involved in genome conformation dynamics could contribute to the discovery of novel antimalarial strategies.
\end{abstract}

{\sc Running title}: Genome architecture of the \emph{P. falciparum} genome
% 50-character limit

\noindent
{\sc Keywords}: malaria, genome architecture, gene expression, virulence genes, erythrocytic cycle\\[3ex]

\clearpage
\input{body}

%
\input{methods}


\paragraph{Data Access}
The sequences and processed data files are deposited to GEO and are available under accession number GSE50199. Additional supplementary data is available at \url{http://noble.gs.washington.edu/proj/plasmo3d}.

\paragraph{Acknowledgements}
We thank Reza Kalhor for advice on the Hi-C protocol and Carolina Borsoi Moraes for advice on the FISH protocol. We thank Harianto Tjong for sharing the source code for volume exclusion simulation on yeast genome. We thank John Weger, Rebecca Sun, and Glenn Hicks (Institute for Integrative Genome Biology, University of California Riverside) for their assistance in the library preparation and sequencing process. We thank Serena Cervantes for technical assistance in microscopy and image processing. The following reagents were obtained through the MR4 as part of the BEI Resources Repository, NIAID, NIH: \emph{Plasmodium falciparum} strain 3D7 (MRA-102) deposited by D.J. Carucci. We thank Stanley Fields and Zhi-jun Duan for feedback on the manuscript. This study was financially supported by a Computing Research Association CIFellows award (NSF award CIF 1136996 to FA), the Human Frontier Science Program (grant LT000507/2011-L to EMB), the National Institutes of Health (grants R01 AI85077-01A1 to KLR and R01 AI106775-01 to WSN/KLR), the European Research Council (grant SMAC-ERC-280032 to JPV), the European Commission (grant HEALTH-F5-2012-305626 to JPV/NV) and the French National Research Agency (grant ANR-11-BINF-0001 to JPV/NV).

\paragraph{Author contributions}
W.S.N. and K.L.R. conceived the project. E.M.B. and S.M.B. carried out the Hi-C and FISH experiments. J.P. maintained {\em P. falciparum} cultures and assisted in experimental procedures. F.A. and N.V. analyzed the data under the supervision of W.S.N. and J.V.\ F.A., E.M.B., N.V., W.S.N. and K.L.R. wrote the manuscript. All authors discussed and edited the manuscript.

\paragraph{Disclosure Declaration}
The authors report no conflict of interest.


\clearpage
\section*{Figure Legends}

\begin{figure}[h]
\centering
\caption{{\bf Tethered conformation capture of the {\em Plasmodium falciparum} genome.}
\textbf{a,} Experimental protocol. \textbf{b,} Contact probability as a function of genomic distance, with log-linear fits for the three erythrocytic stages, as well as an experimental control. \textbf{c,} Normalized contact count matrices at 10 kb resolution for chromosome 2 and chromosome 7 in the schizont stage. \textbf{d,} Contact p-values (negative log10 scale) for chromosome 2 and chromosome 7 in the schizont stage. In \textbf{(c)} and \textbf{(d)}, yellow boxes denote clusters of VRSM genes, and blue dashed lines indicate the centromere location.}
\label{fig:fig1}
\end{figure}

\begin{figure}[h]
\centering
\caption{{\bf 3D modeling and validation with DNA FISH.}
 \textbf{a,} 3D structures of all three stages. The nuclear radii used to model ring, trophozoite and schizont stages were 350, 850, and 425 nm, respectively. Centromeres and telomeres are indicated with light blue and white spheres, respectively. Midpoints of VRSM gene clusters are shown with green spheres. \textbf{b,} Validation of colocalization between a pair of interchromosomal loci with VRSM genes (chr7: 550,000 - 560,000 that harbors internal VRSM genes and chr8: 40,000 - 50,000 that harbors subtelomeric VRSM genes) by DNA FISH (left) and by the three-dimensional model for the corresponding stage (right). The location of the loci in the 3D model is indicated with light blue spheres and pointed by black arrows. \textbf{c,} Validation same as in \textbf{(b)} for a pair of interchromosomal loci that harbor no VRSM genes (chr7: 810,000 - 820,000 and chr11: 820,000 - 830,000).
}
\label{fig:fig2}
\end{figure}

\begin{figure}[h]
\centering
\caption{{\bf Colocalization of highly transcribed rDNA units.}
 Virtual 4C plots generated at 25 kb resolution using as a bait the A-type rDNA unit on chromosome 7 from crosslinked Hi-C libraries of \textbf{(a)} ring, \textbf{(b)} trophozoite, \textbf{(c)} schizont stages and \textbf{(d)} from the trophozoite control library. Vertical red line indicates the midpoint of the A-type rDNA unit on chromosome 5. Normalized contact counts from 50 kb up- and downstream of the 25 kb bin containing the rDNA unit are used, omitting the rDNA-containing window itself to exclude repetitive DNA. For each window $w$ on chromosome 5, the contact enrichment is calculated by dividing the contact count between the bait and $w$ to the average interchromosomal contact count for the bait locus.
}
\label{fig:fig3}
\end{figure}

\begin{figure}[h]
\centering
\caption{{\bf Volume exclusion modeling. }
Observed/expected contact frequency matrices illustrate, for each locus, either the depletion (blue) or enrichment (red) of interaction frequencies compared to what would be expected given their genomic distances. \textbf{a,} Observed/expected contact frequency matrices derived from {\em S. cerevisiae} chr 7 from volume exclusion modeling (left) and Hi-C data (right). \textbf{b,} Observed/expected matrices from volume exclusion modeling (left) and Hi-C data (right) for {\em P. falciparum} chr 7 during the trophozoite stage.}
\label{fig:fig4}
\end{figure}


\begin{figure}[h]
\centering
\caption{{\bf Role of internal VRSM gene clusters in shaping genome architecture.}
 \textbf{a-d,} Heatmaps of scaled pairwise Euclidean distances derived from the 3D model at 10 kb resolution for \textbf{(a, b)} two chromosomes that harbor internal VRSM gene clusters and \textbf{(c, d)} two chromosomes that do not. Yellow boxes indicate locations of VRSM clusters. }
\label{fig:fig5}
\end{figure}

 
\begin{figure}[h]
\centering
\caption{{\bf Relationship between 3D architecture and gene expression.}
\textbf{a,}Correlation between expression profiles of pairs of interchromosomal genes as a function of number of contacts linking the two genes. To generate this plot all interchromosomal gene pairs are first sorted in increasing order of their expression correlation and then binned into 20 equal width quantiles ($5$th, $10$th, ..., $100$th). For each bin, the average expression correlation between gene pairs (x-axis) and the average normalized contact count linking the genes in each pair together with its standard error (y-axis) are computed and plotted. Interchromosomal gene pairs that have contact counts within the top 20\% for each stage have more highly correlated expression profiles than the remaining gene pairs [Wilcoxon rank-sum test, p-values 2.48e-206 (ring), 0 (trophozoite), and 0 (schizont)].
\textbf{b,} Correlation between expression profiles of pairs of interchromosomal genes as a function of 3D distance between the genes. This plot is generated similar to \textbf{a} but with using 3D distances instead of contact counts (y-axis). In order to summarize results from multiple 3D structures per each stage, we plot the median value among 100 structures with a red line and shaded the region corresponding to the interval between 5th and 95th percentile with gray. Interchromosomal gene pairs closer than 20\% of the nuclear diameter have more highly correlated expression profiles than genes that are far apart [Wilcoxon rank-sum test, p-values 7.17e-221 (ring), 0 (trophozoite), and 1.57e-88 (schizont)].
\textbf{c,} Gene expression as a function of distance to telomeres. To generate this plot all genes are first sorted by increasing distance to the centroid of telomeres (x-axis) and then binned similar to \textbf{a} into 20 equal width quantiles. The average log expression value~\citep{bunnik:polysome}
together with its standard error (y-axis) is plotted for genes in each bin. In order to summarize results from multiple 3D structures per each stage, we plot the median value among 100 structures with a red line and shaded the region corresponding to the interval between 5th and 95th percentile with gray. Genes that lie within 20\% of the nuclear diameter to the centroid of the telomeres showed significantly lower expression levels [Wilcoxon rank-sum test, p-values 1.54e-12 (ring), 1.69e-32 (trophozoite), 3.37e-20 (schizont)].
\textbf{d,} First kCCA expression profile component score, corresponding to the projection of the gene expression profile onto the extracted kCCA profile for the trophozoite stage.}
\label{fig:fig6}
\end{figure}

\clearpage


%http://www.biology.emory.edu/Antia/pjohnson/bst/genome_research.bst
\bibliographystyle{genomeresearch}
%\bibliography{refos,refs}
\bibliography{references} 
\end{document}
